apc/proxy.php

```diff
@@ -9,12 +9,35 @@
 	}
 }
 
+/**
+ * Determines if SSL is used.
+ *
+ * @see is_ssl() (wp-includes/load.php).
+ *
+ * @return bool True if SSL, otherwise false.
+ */
+function cachify_is_ssl() {
+	if ( isset( $_SERVER['HTTPS'] ) ) {
+		if ( 'on' == strtolower( $_SERVER['HTTPS'] ) ) {
```

Comment:
**tl;dr**  We need a third opinion to decide.

> The strict comparison is much faster 

Yes it is what's propagated all over the web and true in many cases, but not always. Run a simple (primitive) benchmark with strings yourself:
<details> 
<summary>the script</summary>
<pre>&lt;?php
$test = 'foobar';
$max = 100000000;

$start = microtime(true);
for($i = 0; $i < $max; $i++) {
  if('foobar' == $test) {};
}
$end = microtime(true);
print 'matching     ==  ' . ($end-$start) . "\n";

$start = microtime(true);
for($i = 0; $i < $max; $i++) {
  if('foobar' === $test) {}
}
$end = microtime(true);
print 'matching     === ' . ($end-$start) . "\n";

$start = microtime(true);
for($i = 0; $i < $max; $i++) {
          if('barfoo' != $test) {};
}
$end = microtime(true);
print 'matching     !=  ' . ($end-$start) . "\n";

$start = microtime(true);
for($i = 0; $i < $max; $i++) {
          if('barfoo' !== $test) {}
}
$end = microtime(true);
print 'matching     !== ' . ($end-$start) . "\n";

$start = microtime(true);
for($i = 0; $i < $max; $i++) {
  if('barfoo' == $test) {}
}
$end = microtime(true);
print 'non-matching ==  ' . ($end-$start) . "\n";

$start = microtime(true);
for($i = 0; $i < $max; $i++) {
  if('barfoo' === $test) {}
}
$end = microtime(true);
print 'non-matching === ' . ($end-$start) . "\n";

$start = microtime(true);
for($i = 0; $i < $max; $i++) {
  if('foobar' != $test) {}
}
$end = microtime(true);
print 'non-matching !=  ' . ($end-$start) . "\n";

$start = microtime(true);
for($i = 0; $i < $max; $i++) {
  if('foobar' !== $test) {}
}
$end = microtime(true);
print 'non-matching !== ' . ($end-$start) . "\n";</pre>
</details>

My results (PHP 7.0.21 CLI, 100mio runs) show a tony difference in the opposite direction
```
matching     ==  1.2242779731751
matching     === 1.3983459472656
matching     !=  1.5629131793976
matching     !== 1.7631139755249
non-matching ==  1.7444911003113
non-matching === 1.8354380130768
non-matching !=  1.1368229389191
non-matching !== 1.5221791267395
```
>  if we can get sure, that the string is right, then we should use the strict comparison

If we don't trust the output of a PHP core method, strict comparison is our least problem. The input is not user provided here.

Please don't get me wrong, I don't want to argue about peanuts here, but from my experience, a clean "clone management" has much more benefit for maintaining integrity than adapting everything to own conventions.